tests/test_predict.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import numpy as np
+from pathlib import Path
+
+import src.predict as pr
+import src.file_readers as rd
+
+
+def test_load_model():
+    """
+    Tests load_model function, which loads the silero TTS model.
+    """
+    model = pr.load_model()
+
+    assert str(type(model)) == "<class '<torch_package_0>.multi_acc_v3_package.TTSModelMultiAcc_v3'>"
+    assert model.speakers[0] == 'en_0'
+    assert np.shape(model.speakers) == (119,)
+
+
+def test_generate_audio():
+    """
+    Tests generate_audio function, which takes the TTS model and file input,
+    and uses the predict & write_audio functions to output the audio file.
+    """
+    ebook_path = "tests/data/test.epub"
```

Comment:
Hardcoded paths. Don't do this. Use configs for the folder part. See other comments.

Reply:
Fixed this pathing using pathlib instead of hardcoding.